src/main/java/io/candydoc/ddd/model/UbiquitousLanguage.java

```diff
@@ -0,0 +1,133 @@
+package io.candydoc.ddd.model;
+
+import io.candydoc.ddd.extract_domain_vocabulary.NestedStrategicPatternsForbidden;
+import io.candydoc.ddd.strategic_pattern.BoundedContext;
+import io.candydoc.ddd.strategic_pattern.SharedKernel;
+import io.candydoc.ddd.strategic_pattern.StrategicModel;
+import java.util.Collection;
+import java.util.HashSet;
+import java.util.Set;
+import java.util.stream.Collectors;
+import lombok.Builder;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.ToString;
+import lombok.experimental.Accessors;
+
+public interface UbiquitousLanguage {
+
+  Collection<DomainModel> domainModels();
+
+  static Set<UbiquitousLanguage> fromDomainObjects(FoundDomainObjects foundDomainObjects) {
+    foundDomainObjects.checkConsistency();
+
+    Set<DomainModel> domainModels = foundDomainObjects.toDomainModels();
+    Set<DomainModel> modelsToProcess = new HashSet<>(domainModels);
+    Set<UbiquitousLanguage> ubiquitousLanguages = new HashSet<>();
+
+    domainModels.stream()
+        .filter(StrategicModel.class::isInstance)
+        .map(StrategicModel.class::cast)
+        .forEach(
```

Comment:
Instead of a `Set<DomainModel>`, I think it will be interesting to have some kind of `DomainModels` to handle the stream and hide the cast for StrategicModel:

```java
domainModels.strategic().forEach(...);
```